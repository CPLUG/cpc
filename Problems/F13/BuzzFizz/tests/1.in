20
2
5 Fizz
3 Buzz
