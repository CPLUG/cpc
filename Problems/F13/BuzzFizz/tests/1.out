1
2
Buzz
4
Fizz
Buzz
7
8
Buzz
Fizz
11
Buzz
13
14
BuzzFizz
16
17
Buzz
19
Fizz
